include/language-factory.php

```diff
@@ -41,19 +41,21 @@ public static function get( $language_data ) {
 	 *
 	 * @since 3.4
 	 *
-	 * @param WP_Term[] $terms List of language terms, with the language taxonomy names as array keys.
-	 *                         `language` and `term_language` are mandatory keys.
+	 * @param WP_Term[] $terms   List of language terms, with the language taxonomy names as array keys.
+	 *                           `language` and `term_language` are mandatory keys.
+	 * @param array     $options Array of Polylang's options.
 	 * @return PLL_Language
 	 *
 	 * @phpstan-param array{language:WP_Term, term_language:WP_Term}&array<string, WP_Term> $terms
 	 */
-	public static function get_from_terms( array $terms ) {
+	public static function get_from_terms( array $terms, array $options ) {
```

Comment:
I would have passed `$options`  as dependence of the class in a constructor (mainly because this is how we use options in the other classes), but why not like this? @Screenfeed your opinion?

Reply:
It makes sense. That gives us an excuse to make all methods non-static, for the small cost of it being a little bit more verbose.